Engine fails to compile in grpc dependency.

This is deep down in grpc and appears to be fixed by the patch here released in tag v.1.42.2: https://github.com/grpc/grpc/commit/de6255941a5e1c2fb2d50e57f84e38c09f45023d#diff-fb5da4886fc7000b3572debe8c4b19e6

The error looks like so:
```
 ./pants -V
   Compiling grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)
   ...
error: failed to run custom build command for `grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)`

Caused by:
  process didn't exit successfully: `/home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-e373f3f899a1c091/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=grpc_wrap.c
cargo:rerun-if-changed=grpc
cargo:rerun-if-env-changed=GRPCIO_SYS_USE_PKG_CONFIG
running: "cmake" "/home/jsirois/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc" "-DCMAKE_INSTALL_PREFIX=/home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-5481e0a69973bf20/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/g++" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-5481e0a69973bf20/out/build
running: "cmake" "--build" "." "--target" "grpc" "--config" "RelWithDebInfo" "--"
[  0%] Built target pkcs8_lib
[  0%] Built target stack
[  0%] Built target lhash
[  2%] Built target err
[  2%] Built target buf
[  2%] Built target base64
[  5%] Built target bytestring
[  5%] Built target pool
[  8%] Built target sha
[  8%] Built target md4
[  8%] Built target md5
[  8%] Built target modes
[  8%] Built target aes
[  8%] Built target des
[  8%] Built target rc4
[  8%] Built target conf
[  8%] Built target chacha
[  8%] Built target poly1305
[  8%] Built target curve25519
[  8%] Built target digest
[ 11%] Built target cipher
[ 14%] Built target rand
[ 14%] Built target bio
[ 17%] Built target bn
[ 17%] Built target obj
[ 20%] Built target asn1
[ 20%] Built target engine
[ 20%] Built target dh
[ 20%] Built target dsa
[ 20%] Built target rsa
[ 23%] Built target ec
[ 23%] Built target ecdh
[ 23%] Built target ecdsa
[ 23%] Built target hmac
[ 23%] Built target cmac
[ 23%] Built target evp
[ 23%] Built target hkdf
[ 23%] Built target pem
[ 32%] Built target x509
[ 38%] Built target x509v3
[ 38%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/log_linux.c.o

--- stderr
CMake Warning at CMakeLists.txt:122 (message):
  gRPC_INSTALL will be forced to FALSE because gRPC_ZLIB_PROVIDER is "module"


-- git Version: v1.1.0-pre3-d00c06b3
-- Version: 1.1.0
-- Performing Test HAVE_STD_REGEX
-- Performing Test HAVE_STD_REGEX -- success
-- Performing Test HAVE_GNU_POSIX_REGEX
-- Performing Test HAVE_GNU_POSIX_REGEX -- failed to compile
-- Performing Test HAVE_POSIX_REGEX
-- Performing Test HAVE_POSIX_REGEX -- success
-- Performing Test HAVE_STEADY_CLOCK
-- Performing Test HAVE_STEADY_CLOCK -- success
/home/jsirois/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc/src/core/lib/support/log_linux.c:42:13: error: conflicting types for ‘gettid’
   42 | static long gettid(void) { return syscall(__NR_gettid); }
      |             ^~~~~~
In file included from /usr/include/unistd.h:1170,
                 from /home/jsirois/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc/src/core/lib/support/log_linux.c:40:
/usr/include/bits/unistd_ext.h:34:16: note: previous declaration of ‘gettid’ was here
   34 | extern __pid_t gettid (void) __THROW;
      |                ^~~~~~
make[3]: *** [CMakeFiles/gpr.dir/build.make:543: CMakeFiles/gpr.dir/src/core/lib/support/log_linux.c.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:592: CMakeFiles/gpr.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:567: CMakeFiles/grpc.dir/rule] Error 2
make: *** [Makefile:320: grpc] Error 2
thread 'main' panicked at '
command did not execute successfully, got: exit code: 2

build script failed, must exit now', /home/jsirois/.cache/pants/rust/cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.40/src/lib.rs:832:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

Failed to build native engine.
```

I'm using gcc 9.2.0:
```
$ gcc --version
gcc (GCC) 9.2.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Switching to clang:
```
$ CC=clang ./pants -V
   Compiling grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)
   ...
(/home/jsirois/dev/pantsbuild/pants/src/rust/engine/process_execution/bazel_protos)
error: failed to run custom build command for `grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)`

Caused by:
  process didn't exit successfully: `/home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-e373f3f899a1c091/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=grpc_wrap.c
cargo:rerun-if-changed=grpc
cargo:rerun-if-env-changed=GRPCIO_SYS_USE_PKG_CONFIG
running: "cmake" "/home/jsirois/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc" "-DCMAKE_INSTALL_PREFIX=/home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-5481e0a69973bf20/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC --target=x86_64-unknown-linux-gnu" "-DCMAKE_C_COMPILER=/usr/bin/clang" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/g++" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/jsirois/dev/pantsbuild/pants/src/rust/engine/target/release/build/grpcio-sys-5481e0a69973bf20/out/build
running: "cmake" "--build" "." "--target" "grpc" "--config" "RelWithDebInfo" "--"
[  0%] Building C object third_party/boringssl/crypto/stack/CMakeFiles/stack.dir/stack.c.o
[  0%] Building C object third_party/boringssl/crypto/lhash/CMakeFiles/lhash.dir/lhash.c.o
[  0%] Building C object third_party/boringssl/crypto/err/CMakeFiles/err.dir/err.c.o
[  0%] Building C object third_party/boringssl/crypto/buf/CMakeFiles/buf.dir/buf.c.o
[  0%] Building C object third_party/boringssl/crypto/pkcs8/CMakeFiles/pkcs8_lib.dir/pkcs8.c.o
[  0%] Built target buf
[  0%] Built target stack
[  0%] Building C object third_party/boringssl/crypto/base64/CMakeFiles/base64.dir/base64.c.o
[  0%] Built target lhash
[  0%] Building C object third_party/boringssl/crypto/pkcs8/CMakeFiles/pkcs8_lib.dir/p8_pkey.c.o
[  0%] Building C object third_party/boringssl/crypto/pkcs8/CMakeFiles/pkcs8_lib.dir/p5_pbev2.c.o
[  2%] Building C object third_party/boringssl/crypto/err/CMakeFiles/err.dir/err_data.c.o
[  2%] Built target err
[  2%] Building C object third_party/boringssl/crypto/bytestring/CMakeFiles/bytestring.dir/asn1_compat.c.o
[  2%] Building C object third_party/boringssl/crypto/bytestring/CMakeFiles/bytestring.dir/ber.c.o
[  5%] Building C object third_party/boringssl/crypto/bytestring/CMakeFiles/bytestring.dir/cbs.c.o
[  5%] Built target pkcs8_lib
[  5%] Built target base64
[  5%] Building C object third_party/boringssl/crypto/pool/CMakeFiles/pool.dir/pool.c.o
[  5%] Building C object third_party/boringssl/crypto/sha/CMakeFiles/sha.dir/sha1-altivec.c.o
[  5%] Building C object third_party/boringssl/crypto/bytestring/CMakeFiles/bytestring.dir/cbb.c.o
[  5%] Building C object third_party/boringssl/crypto/sha/CMakeFiles/sha.dir/sha1.c.o
[  5%] Building C object third_party/boringssl/crypto/sha/CMakeFiles/sha.dir/sha256.c.o
[  5%] Built target pool
[  8%] Building C object third_party/boringssl/crypto/sha/CMakeFiles/sha.dir/sha512.c.o
[  8%] Built target bytestring
[  8%] Building C object third_party/boringssl/crypto/md4/CMakeFiles/md4.dir/md4.c.o
[  8%] Building C object third_party/boringssl/crypto/md5/CMakeFiles/md5.dir/md5.c.o
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/cbc.c.o
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/cfb.c.o
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/ctr.c.o
[  8%] Built target md5
[  8%] Built target md4
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/gcm.c.o
[  8%] Building C object third_party/boringssl/crypto/aes/CMakeFiles/aes.dir/aes.c.o
[  8%] Building C object third_party/boringssl/crypto/des/CMakeFiles/des.dir/des.c.o
[  8%] Building C object third_party/boringssl/crypto/rc4/CMakeFiles/rc4.dir/rc4.c.o
[  8%] Built target rc4
[  8%] Building C object third_party/boringssl/crypto/conf/CMakeFiles/conf.dir/conf.c.o
[  8%] Building C object third_party/boringssl/crypto/aes/CMakeFiles/aes.dir/key_wrap.c.o
[  8%] Built target sha
[  8%] Building C object third_party/boringssl/crypto/aes/CMakeFiles/aes.dir/mode_wrappers.c.o
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/ofb.c.o
[  8%] Built target aes
[  8%] Building C object third_party/boringssl/crypto/modes/CMakeFiles/modes.dir/polyval.c.o
[  8%] Built target des
[  8%] Built target conf
[  8%] Building C object third_party/boringssl/crypto/chacha/CMakeFiles/chacha.dir/chacha.c.o
[  8%] Building C object third_party/boringssl/crypto/poly1305/CMakeFiles/poly1305.dir/poly1305.c.o
[  8%] Building C object third_party/boringssl/crypto/curve25519/CMakeFiles/curve25519.dir/curve25519.c.o
[  8%] Building C object third_party/boringssl/crypto/digest/CMakeFiles/digest.dir/digest.c.o
[  8%] Built target modes
[  8%] Building C object third_party/boringssl/crypto/digest/CMakeFiles/digest.dir/digests.c.o
[  8%] Building C object third_party/boringssl/crypto/poly1305/CMakeFiles/poly1305.dir/poly1305_vec.c.o
[  8%] Building C object third_party/boringssl/crypto/poly1305/CMakeFiles/poly1305.dir/poly1305_arm.c.o
[  8%] Built target chacha
[  8%] Building C object third_party/boringssl/crypto/curve25519/CMakeFiles/curve25519.dir/spake25519.c.o
[  8%] Built target digest
[  8%] Building C object third_party/boringssl/crypto/curve25519/CMakeFiles/curve25519.dir/x25519-x86_64.c.o
[ 11%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/cipher.c.o
[ 11%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/derive_key.c.o
[ 11%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/aead.c.o
[ 11%] Building C object third_party/boringssl/crypto/rand/CMakeFiles/rand.dir/deterministic.c.o
[ 11%] Building C object third_party/boringssl/crypto/rand/CMakeFiles/rand.dir/fuchsia.c.o
[ 11%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_null.c.o
[ 11%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_rc2.c.o
[ 11%] Building C object third_party/boringssl/crypto/rand/CMakeFiles/rand.dir/rand.c.o
[ 11%] Building C object third_party/boringssl/crypto/rand/CMakeFiles/rand.dir/urandom.c.o
[ 11%] Built target poly1305
[ 14%] Building C object third_party/boringssl/crypto/rand/CMakeFiles/rand.dir/windows.c.o
[ 14%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/bio.c.o
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_rc4.c.o
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_des.c.o
[ 14%] Built target rand
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_aes.c.o
[ 14%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/bio_mem.c.o
[ 14%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/add.c.o
[ 14%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/asm/x86_64-gcc.c.o
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_chacha20poly1305.c.o
[ 14%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/bn.c.o
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/tls_cbc.c.o
[ 14%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/bn_asn1.c.o
[ 14%] Built target curve25519
[ 14%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_tls.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/cmp.c.o
[ 17%] Building C object third_party/boringssl/crypto/obj/CMakeFiles/obj.dir/obj.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/convert.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/ctx.c.o
[ 17%] Building C object third_party/boringssl/crypto/obj/CMakeFiles/obj.dir/obj_xref.c.o
[ 17%] Built target obj
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_bitstr.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/div.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_bool.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/exponentiation.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_d2i_fp.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_dup.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_enum.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/generic.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_gentm.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_i2d_fp.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_int.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_mbstr.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_object.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/gcd.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_octet.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_print.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/kronecker.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_strnid.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/montgomery.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_time.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/montgomery_inv.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_type.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/mul.c.o
[ 17%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_utctm.c.o
[ 17%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/prime.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/a_utf8.c.o
[ 20%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/random.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/asn1_lib.c.o
[ 20%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/shift.c.o
[ 20%] Building C object third_party/boringssl/crypto/bn/CMakeFiles/bn.dir/sqrt.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/asn1_par.c.o
[ 20%] Built target bn
[ 20%] Building C object third_party/boringssl/crypto/engine/CMakeFiles/engine.dir/engine.c.o
[ 20%] Built target engine
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/asn_pack.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/connect.c.o
[ 20%] Building C object third_party/boringssl/crypto/dh/CMakeFiles/dh.dir/dh.c.o
[ 20%] Building C object third_party/boringssl/crypto/dsa/CMakeFiles/dsa.dir/dsa.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/fd.c.o
[ 20%] Building C object third_party/boringssl/crypto/dh/CMakeFiles/dh.dir/params.c.o
[ 20%] Building C object third_party/boringssl/crypto/dh/CMakeFiles/dh.dir/check.c.o
[ 20%] Building C object third_party/boringssl/crypto/dsa/CMakeFiles/dsa.dir/dsa_asn1.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/file.c.o
[ 20%] Building C object third_party/boringssl/crypto/dh/CMakeFiles/dh.dir/dh_asn1.c.o
[ 20%] Built target dsa
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/f_enum.c.o
[ 20%] Built target dh
[ 20%] Building C object third_party/boringssl/crypto/cipher/CMakeFiles/cipher.dir/e_ssl3.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/hexdump.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/f_int.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/f_string.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/t_bitst.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_dec.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/pair.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/printf.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_enc.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_fre.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/socket.c.o
[ 20%] Building C object third_party/boringssl/crypto/bio/CMakeFiles/bio.dir/socket_helper.c.o
[ 20%] Built target cipher
[ 20%] Building C object third_party/boringssl/crypto/rsa/CMakeFiles/rsa.dir/rsa.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/ec.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_new.c.o
[ 20%] Building C object third_party/boringssl/crypto/rsa/CMakeFiles/rsa.dir/rsa_impl.c.o
[ 20%] Built target bio
[ 20%] Building C object third_party/boringssl/crypto/rsa/CMakeFiles/rsa.dir/blinding.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_typ.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/tasn_utl.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/ec_asn1.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/ec_key.c.o
[ 20%] Building C object third_party/boringssl/crypto/rsa/CMakeFiles/rsa.dir/padding.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/ec_montgomery.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/time_support.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/x_bignum.c.o
[ 20%] Building C object third_party/boringssl/crypto/asn1/CMakeFiles/asn1.dir/x_long.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/oct.c.o
[ 20%] Building C object third_party/boringssl/crypto/rsa/CMakeFiles/rsa.dir/rsa_asn1.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/p224-64.c.o
[ 20%] Built target asn1
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/p256-64.c.o
[ 20%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/p256-x86_64.c.o
[ 20%] Building C object third_party/boringssl/crypto/ecdh/CMakeFiles/ecdh.dir/ecdh.c.o
[ 20%] Built target rsa
[ 23%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/simple.c.o
[ 23%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/util-64.c.o
[ 23%] Built target ecdh
[ 23%] Building C object third_party/boringssl/crypto/ec/CMakeFiles/ec.dir/wnaf.c.o
[ 23%] Building C object third_party/boringssl/crypto/ecdsa/CMakeFiles/ecdsa.dir/ecdsa.c.o
[ 23%] Building C object third_party/boringssl/crypto/ecdsa/CMakeFiles/ecdsa.dir/ecdsa_asn1.c.o
[ 23%] Building C object third_party/boringssl/crypto/hmac/CMakeFiles/hmac.dir/hmac.c.o
[ 23%] Building C object third_party/boringssl/crypto/cmac/CMakeFiles/cmac.dir/cmac.c.o
[ 23%] Built target ecdsa
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/digestsign.c.o
[ 23%] Built target hmac
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/evp.c.o
[ 23%] Built target cmac
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/evp_asn1.c.o
[ 23%] Building C object third_party/boringssl/crypto/hkdf/CMakeFiles/hkdf.dir/hkdf.c.o
[ 23%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_all.c.o
[ 23%] Built target hkdf
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/evp_ctx.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/p_dsa_asn1.c.o
[ 23%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_info.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/p_ec.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/p_ec_asn1.c.o
[ 23%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_lib.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/p_rsa.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/p_rsa_asn1.c.o
[ 23%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/a_digest.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/pbkdf.c.o
[ 23%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/a_sign.c.o
[ 23%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/print.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/a_strex.c.o
[ 26%] Building C object third_party/boringssl/crypto/evp/CMakeFiles/evp.dir/sign.c.o
[ 26%] Built target evp
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/a_verify.c.o
[ 26%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_oth.c.o
[ 26%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_pk8.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/algorithm.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/asn1_gen.c.o
[ 26%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_pkey.c.o
[ 26%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_x509.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/by_dir.c.o
[ 26%] Building C object third_party/boringssl/crypto/pem/CMakeFiles/pem.dir/pem_xaux.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_cache.c.o
[ 26%] Built target pem
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_data.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/i2d_pr.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/by_file.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_lib.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/pkcs7.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/rsa_pss.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_map.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_node.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/t_crl.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/pcy_tree.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/t_req.c.o
[ 26%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/t_x509.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_akey.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/t_x509a.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_akeya.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/profiling/basic_timers.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_alt.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/profiling/stap_timers.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_att.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/alloc.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_bcons.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/arena.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_cmp.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/atm.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_bitst.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/avl.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_conf.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_d2.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/backoff.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_def.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_cpols.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/cmdline.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_ext.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/cpu_iphone.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_crld.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_lu.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/cpu_linux.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/cpu_posix.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/cpu_windows.c.o
[ 29%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_enum.c.o
[ 29%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/env_linux.c.o
[ 32%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_obj.c.o
[ 32%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_extku.c.o
[ 32%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/env_posix.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/env_windows.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_r2x.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_req.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_genn.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/fork.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_ia5.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_set.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/histogram.c.o
[ 35%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/adler32.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_info.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_trs.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/host_port.c.o
[ 35%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/compress.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_int.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/log.c.o
[ 35%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/crc32.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_txt.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_lib.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/log_android.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_v3.c.o
[ 35%] Building C object CMakeFiles/gpr.dir/src/core/lib/support/log_linux.c.o
[ 35%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/deflate.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_ncons.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_pci.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_vfy.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509_vpm.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_pcia.c.o
[ 35%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_pcons.c.o
[ 35%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/gzclose.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_pku.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_pmaps.c.o
[ 38%] Built target ec
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_prn.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509cset.c.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/gzlib.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/gzread.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509name.c.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/gzwrite.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/inflate.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509rset.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_purp.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509spki.c.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/infback.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x509type.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_algor.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_all.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_attrib.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_skey.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_sxnet.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509v3/CMakeFiles/x509v3.dir/v3_utl.c.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/inftrees.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_crl.c.o
[ 38%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_exten.c.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/inffast.o
[ 38%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/trees.o
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_info.c.o
[ 41%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/uncompr.o
[ 41%] Building C object third_party/zlib/CMakeFiles/zlibstatic.dir/zutil.o
[ 41%] Built target x509v3
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_name.c.o
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_pkey.c.o
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_pubkey.c.o
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_req.c.o
[ 41%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_sig.c.o
[ 44%] Linking C static library libz.a
[ 44%] Built target zlibstatic
[ 44%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_spki.c.o
[ 44%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_x509.c.o
[ 44%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_val.c.o
[ 44%] Building C object third_party/boringssl/crypto/x509/CMakeFiles/x509.dir/x_x509a.c.o
[ 44%] Built target x509

--- stderr
CMake Warning at CMakeLists.txt:122 (message):
  gRPC_INSTALL will be forced to FALSE because gRPC_ZLIB_PROVIDER is "module"


-- git Version: v1.1.0-pre3-d00c06b3
-- Version: 1.1.0
-- Performing Test HAVE_STD_REGEX
-- Performing Test HAVE_STD_REGEX -- success
-- Performing Test HAVE_GNU_POSIX_REGEX
-- Performing Test HAVE_GNU_POSIX_REGEX -- failed to compile
-- Performing Test HAVE_POSIX_REGEX
-- Performing Test HAVE_POSIX_REGEX -- success
-- Performing Test HAVE_STEADY_CLOCK
-- Performing Test HAVE_STEADY_CLOCK -- success
/home/jsirois/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc/src/core/lib/support/log_linux.c:42:13: error: static declaration of 'gettid' follows non-static declaration
static long gettid(void) { return syscall(__NR_gettid); }
            ^
/usr/include/bits/unistd_ext.h:34:16: note: previous declaration is here
extern __pid_t gettid (void) __THROW;
               ^
1 error generated.
make[3]: *** [CMakeFiles/gpr.dir/build.make:543: CMakeFiles/gpr.dir/src/core/lib/support/log_linux.c.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:592: CMakeFiles/gpr.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:567: CMakeFiles/grpc.dir/rule] Error 2
make: *** [Makefile:320: grpc] Error 2
thread 'main' panicked at '
command did not execute successfully, got: exit code: 2

build script failed, must exit now', /home/jsirois/.cache/pants/rust/cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.40/src/lib.rs:832:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

Failed to build native engine.
```

I'm using clang 9.0.0:
```
$ clang --version
clang version 9.0.0 (tags/RELEASE_900/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

Same / similar issue on Mac OS Catalina with `~/.cache/pants` dir nuked at 725d5d5b9aed59a800eaa1b60f3e488c95f8c8c4 
```
./pants goals
   Compiling grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)
   Compiling engine_cffi v0.0.1 (/Users/rjiang/workspace/pants/src/rust/engine/engine_cffi)
   Compiling cargo v0.34.0
   Compiling crossbeam-deque v0.7.1
   Compiling parking_lot v0.6.4
   Compiling tower-grpc-build v0.1.0 (https://github.com/pantsbuild/tower-grpc.git?rev=ef19f2e1715f415ecb699e8f17f5845ad2b45daf#ef19f2e1)
   Compiling tokio-reactor v0.1.9
error: failed to run custom build command for `grpcio-sys v0.2.3 (https://github.com/pantsbuild/grpc-rs.git?rev=9dd357da5a2f231ce254d8abdd46068198637beb#9dd357da)`

Caused by:
  process didn't exit successfully: `/Users/rjiang/workspace/pants/src/rust/engine/target/release/build/grpcio-sys-0239a6bdf86ba4cd/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=grpc_wrap.c
cargo:rerun-if-changed=grpc
cargo:rerun-if-env-changed=GRPCIO_SYS_USE_PKG_CONFIG
running: "cmake" "/Users/rjiang/.cache/pants/rust/cargo/git/checkouts/grpc-rs-b50c79747a664460/9dd357d/grpc-sys/grpc" "-DCMAKE_INSTALL_PREFIX=/Users/rjiang/workspace/pants/src/rust/engine/target/release/build/grpcio-sys-51d79ce600b46b11/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -stdlib=libc++ -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/rjiang/workspace/pants/src/rust/engine/target/release/build/grpcio-sys-51d79ce600b46b11/out/build
running: "cmake" "--build" "." "--target" "grpc" "--config" "RelWithDebInfo" "--"
[  2%] Built target pkcs8_lib
[  2%] Built target stack
[  2%] Built target lhash
[  2%] Generating err_data.c

--- stderr
CMake Warning at CMakeLists.txt:122 (message):
  gRPC_INSTALL will be forced to FALSE because gRPC_ZLIB_PROVIDER is "module"


-- git Version: v1.1.0-pre3-725d5d5b
-- Version: 1.1.0
-- Performing Test HAVE_THREAD_SAFETY_ATTRIBUTES
-- Performing Test HAVE_THREAD_SAFETY_ATTRIBUTES -- failed to compile
-- Performing Test HAVE_STD_REGEX
-- Performing Test HAVE_STD_REGEX -- success
-- Performing Test HAVE_GNU_POSIX_REGEX
-- Performing Test HAVE_GNU_POSIX_REGEX -- failed to compile
-- Performing Test HAVE_POSIX_REGEX
-- Performing Test HAVE_POSIX_REGEX -- success
-- Performing Test HAVE_STEADY_CLOCK
-- Performing Test HAVE_STEADY_CLOCK -- success
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   grpc_csharp_ext
   zlib

This warning is for project developers.  Use -Wno-dev to suppress it.

go tool asm: exit status 2
flag provided but not defined: -V
usage: asm [options] file.s
Flags:
  -D value
    	predefined symbol with optional simple value -D=identifier=value; can be set multiple times
  -I value
    	include directory; can be set multiple times
  -S	print assembly and machine code
  -debug
    	dump instructions as they are parsed
  -dynlink
    	support references to Go symbols defined in other shared libraries
  -e	no limit on number of errors reported
  -o string
    	output file; default foo.6 for /a/b/c/foo.s on amd64
  -shared
    	generate code that can be linked into a shared library
  -trimpath string
    	remove prefix from recorded source file paths
make[3]: *** [third_party/boringssl/crypto/err/err_data.c] Error 1
make[3]: *** Deleting file `third_party/boringssl/crypto/err/err_data.c'
make[2]: *** [third_party/boringssl/crypto/err/CMakeFiles/err.dir/all] Error 2
make[1]: *** [CMakeFiles/grpc.dir/rule] Error 2
make: *** [grpc] Error 2
thread 'main' panicked at '
command did not execute successfully, got: exit code: 2

build script failed, must exit now', /Users/rjiang/.cache/pants/rust/cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.40/src/lib.rs:832:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

Failed to build native engine.
```

--- a/src/rust/engine/fs/store/Cargo.toml
+++ b/src/rust/engine/fs/store/Cargo.toml
@@ -13,7 +13,7 @@ digest = "0.8"
 dirs = "1"
 fs = { path = ".." }
 futures = "^0.1.16"
-grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "9dd357da5a2f231ce254d8abdd46068198637beb", default_features = false, features = ["protobuf-codec", "secure"] }
+grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "b582ef3dc4e8c7289093c8febff8dadf0997b532", default_features = false, features = ["protobuf-codec", "secure"] }
 hashing = { path = "../../hashing" }
 indexmap = "1.0.2"
 itertools = "0.7.2"

--- a/src/rust/engine/process_execution/Cargo.toml
+++ b/src/rust/engine/process_execution/Cargo.toml
@@ -16,7 +16,7 @@ derivative = "1.0.2"
 digest = "0.8"
 fs = { path = "../fs" }
 futures = "^0.1.16"
-grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "9dd357da5a2f231ce254d8abdd46068198637beb", default_features = false, features = ["protobuf-codec", "secure"] }
+grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "b582ef3dc4e8c7289093c8febff8dadf0997b532", default_features = false, features = ["protobuf-codec", "secure"] }
 hashing = { path = "../hashing" }
 libc = "0.2.39"
 log = "0.4"

--- a/src/rust/engine/process_execution/bazel_protos/Cargo.toml
+++ b/src/rust/engine/process_execution/bazel_protos/Cargo.toml
@@ -8,7 +8,7 @@ publish = false
 [dependencies]
 bytes = "0.4.5"
 futures = "^0.1.16"
-grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "9dd357da5a2f231ce254d8abdd46068198637beb", default_features = false, features = ["protobuf-codec", "secure"] }
+grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "b582ef3dc4e8c7289093c8febff8dadf0997b532", default_features = false, features = ["protobuf-codec", "secure"] }
 hashing = { path = "../../hashing" }
 prost = "0.4"
 prost-derive = "0.4"

--- a/src/rust/engine/testutil/mock/Cargo.toml
+++ b/src/rust/engine/testutil/mock/Cargo.toml
@@ -9,7 +9,7 @@ publish = false
 bazel_protos = { path = "../../process_execution/bazel_protos" }
 bytes = "0.4.5"
 futures = "^0.1.16"
-grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "9dd357da5a2f231ce254d8abdd46068198637beb", default_features = false, features = ["protobuf-codec", "secure"] }
+grpcio = { git = "https://github.com/pantsbuild/grpc-rs.git", rev = "b582ef3dc4e8c7289093c8febff8dadf0997b532", default_features = false, features = ["protobuf-codec", "secure"] }
 hashing = { path = "../../hashing" }
 parking_lot = "0.6"
 protobuf = { version = "2.0.6", features = ["with-bytes"] }

